controllers/pulp/controller.go

```diff
@@ -66,9 +66,10 @@ type PulpReconciler struct {
 // move the current state of the cluster closer to the desired state.
 func (r *PulpReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := r.RawLogger
+
 	IsOpenShift, _ := controllers.IsOpenShift()
 	if IsOpenShift {
-		log.Info("Running on OpenShift cluster")
+		log.V(1).Info("Running on OpenShift cluster")
```

Comment:
what does `V(1)` mean?

Reply:
It overrides the log level to 1 (debug):
https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/#output-overriding-the-log-level-to-1-debug